rocketpy/simulation/flight.py

```diff
@@ -2413,7 +2388,7 @@ def aerodynamic_spin_moment(self):
     def rotational_energy(self):
         # b = -self.rocket.distanceRocketPropellant
         b = (
-            -(self.rocket.motor_position - self.rocket.center_of_dry_mass_position)
+            -(self.rocket.nozzle_position - self.rocket.center_of_dry_mass_position)
```

Comment:
I think this is wrong... See the line 2389 right above... the parameter `b` should be the distance from the rocket's CDM to the propellant center of mass.

The way it is implemented right now, it is always computing a constant distance, instead of a floating one.

Reply:
What is written here is actually the r_nozzle

Reply:
Fixed in https://github.com/RocketPy-Team/RocketPy/pull/423/commits/81e4d1a44a988675b30e89074283da6e932b69dd